pom.xml

```diff
@@ -90,6 +91,12 @@
       <artifactId>slf4j-api</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.fasterxml.uuid</groupId>
+      <artifactId>java-uuid-generator</artifactId>
+      <version>${version.uuid}</version>
+    </dependency>
```

Comment:
:question: Why do you want to use this library?

Java has a built-in `UUID` class. You could generate UUID by using `UUID.randomUUID()`.

Reply:
if you insist on the Java build-in `UUID.randomUUID()`, i can change to it.

Reply:
@skayliu very interesting. :+1: I see that this library is also used by Camunda 7. :bulb: That is a good indication for using it here too.